base/arrayshow.jl

```diff
@@ -202,7 +202,7 @@ function _print_matrix(io, @nospecialize(X::AbstractVecOrMat), pre, sep, post, h
     if n > maxpossiblecols
         colsA = [colsA[(0:maxpossiblecols-1) .+ firstindex(colsA)]; colsA[(end-maxpossiblecols+1):end]]
     else
-	    colsA = [colsA;]
+	colsA = [colsA;]
```

Comment:
```suggestion
        colsA = [colsA;]
```

Reply:
Ooo, it was a tab, not just extra indentation. Well spotted. Perhaps this should be a part of the whitespace check.